view/next-project/src/components/fund_information/AddModal.tsx

```diff
@@ -132,7 +133,14 @@ const OpenAddModal: FC<ModalProps> = (props) => {
         </div>
         <p className='col-span-1 text-black-600'>金額</p>
         <div className='col-span-4 w-full'>
-          <Input className='w-full' value={formData.price} onChange={handler('price')} />
+          <Input
+            className='w-full'
+            value={formData.price}
+            onChange={handler('price')}
+            datalist={DONATION_AMOUNT}
+            listKey='amoutOptions'
+            enableDatalist={true}
```

Comment:
```suggestion
```
上のように修正すれば消せます。

Reply:
ちなみにtrueを渡さなくても
```suggestion
            enableDatalist
```
のみでtrueのように動作します